client/service/src/task_manager.rs

```diff
@@ -188,3 +195,9 @@ impl Drop for TaskManager {
 		}
 	}
 }
+
+#[test]
+fn can_make_dyn_clonable_spawn() {
```

Comment:
Why does this test not lives at the point where we declare the trait?

Reply:
It strayed...